pkg/sql/logictest/testdata/logic_test/column_families

```diff
@@ -79,3 +79,40 @@ ORDER BY message
 fetched: /t/t_pkey/1/2.00/x -> /1.00
 fetched: /t/t_pkey/1/2/y -> /2.00
 fetched: /t/t_pkey/1/2/z -> /1
+
+# Regression test for #131860.
+
+statement ok
+CREATE TABLE abc (a INT NOT NULL, b FLOAT NOT NULL, c INT, FAMILY (a), FAMILY (b), FAMILY (c))
+
+query S kvtrace(prefix=/Table/109)
+INSERT INTO abc VALUES (4, -0, 6)
+----
+CPut /Table/109/1/1010419554747023361/0 -> /TUPLE/1:1:Int/4
```

Comment:
I'm surprised this works for secondary tenant testing.

Reply:
Heh, indeed, it did not. I've tweaked the kvtrace option of logictests to strip out tenant prefixes, so it should work now.